spec.html

```diff
@@ -49342,6 +49342,7 @@ <h1>
           </dl>
           <emu-alg>
             1. Let _values_ be a new empty List.
+            1. [declared="remainingElementsCount"] NOTE: _remainingElementsCount_ starts at 1 instead of 0 to ensure _resultCapability_.[[Resolve]] is only called once, even in the presence of a misbehaving *"then"* which calls the passed callback before the input iterator is exhausted.
```

Comment:
```suggestion
            1. [declared="remainingElementsCount"] NOTE: _remainingElementsCount_ starts at 1 instead of 0 to ensure _resultCapability_.[[Resolve]] is only called once, even in the presence of a misbehaving *"then"* which calls one of the passed callbacks before the input iterator is exhausted.
```

Technically all four of the combinators pass two callbacks, but this is the only one where invoking either one decrements the count (in `all` it's just the first and in `any` it's just the second), so this sentence in those sections can be read to have "the" callback refer to the relevant one. Not so here.